test/data/build.yaml

```diff
@@ -14,3 +14,20 @@ parameters:
 meta:
     coverage: 99.54
 status: SUCCESS
```

Comment:
This really doesn't look like a successful job based on the status codes in steps.